c_secure_comm.c

```diff
@@ -377,8 +503,8 @@ session_key_list_t *send_session_key_request_check_protocol(
     } else if (strcmp((const char *)ctx->config->network_protocol, "UDP") ==
                0) {
         // TODO:(Dongha Kim): Implement session key request via UDP.
```

Comment:
Please create an issue for this TODO.

Reply:
Added here. d2adfae8aa8ee4c09bd331c81d804798dc2f09f1